pkgs/by-name/we/weasis/package.nix

```diff
@@ -2,7 +2,7 @@
   lib,
   stdenv,
   fetchzip,
-  jre,
+  jdk23,
```

Comment:
@TomaSajt Can you let me know if this change is valid?

Reply:
I think this is valid.

The `jre` argument is usually provided, so that people may use a minimal jdk, that only has to minimum required modules.

However, from what I've seen, the `build/script/package-weasis.sh` script itself does the stripping of the provided jdk via `jpackage` (which uses `jlink` internally, which is what nixpkgs uses for minimal JREs).

While this change may change the exact `.override` interface of the package, IMO it's no big deal.

Reply:
Thank you !